package.json

```
@@ -0,0 +1,91 @@
+{
+  "name": "jci-moyeo-design-system",
+  "version": "0.1.1",
+  "module": "dist/index.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "/dist"
+  ],
+  "dependencies": {},
+  "peerDependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "scripts": {
+    "storybook": "start-storybook -p 6006 -s public",
```

```suggestion
    "start": "start-storybook -p 6006 -s public",
```
취향 차이이긴 한데 `start`로 바꾸면 어떨지?ㅋㅋ

---

전 storybook 쓰긴하는데 start도 괜찮습니다 ㅎㅎ